### DISCUSSION TITLE
Block zones for Log Flumes

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#18899] [Feature requests]
I was thinking it'd be cool to make the log flume more advanced and interesting. Already started that by adding steep sprites.
I think it'd be a good addition to have block zones for specific parts of flumes. This means a holding block at the beginning and a marker block for end of the block zone. The rest of the ride would function as if it wouldn't have block brakes. This is also true for the real life counter part so two boats dont float into one another at high speed. Could introduce it that boats explode if they collide with one another at a high enough speed (say the front boat loses speed way faster than the boat behind it and they aren't very far apart).

**Comment by ocalhoun6** [comment]
>Could introduce it that boats explode if they collide with one another at a high enough speed

This could break a lot of existing parks, which were built on the assumption that the log flume never crashes. Especially in situations where people have a high drop leading directly into the station platform, where logs might be backed up if they're not leaving the station fast enough.

**Comment by astraylife1** [comment]
You can already add blocks by changing the ride type to a coaster with blocks. Add the blocks. Change it back to a flume. Just need to use tile inspector to overlay the block pieces with regular flume.

**Reply by karst** [reply]
This isn't what I am looking for. I am looking for a specific ride element that stops boats at a specific holding brake until the specific section has been cleared by one boat. This means you could still have 30 boats running, but boats will be helt at the top of a drop for example until it's been cleared. Meaning the rest of the ride isn't blockzoned. Only a very specific part of the ride. Unlike roller coasters, it's ok for boats to collide, just not at very high speeds.

**Reply by astraylife1** [reply]
Seems like you’d have to introduce this to splash boats and rapids boats as well. Not sure how the game could handle an operating mode of continuous circuit but also block section mode simultaneously.

**Reply by karst** [reply]
I imagine it could work similarly to how cable lifts get calculated.
